riscv_isac/InstructionObject.py

```diff
@@ -72,4 +74,371 @@ def __str__(self):
             line+= ' csr_commit: '+ str(self.csr_commit)
         if self.mnemonic:
             line+= ' mnemonic: '+ str(self.mnemonic)
-        return line
\ No newline at end of file
+        return line
+
+
+    # Put the following function in instructionObject.py 
+    def init_rvp_dictionary(self):

```

Comment:
Please move these functions to the `internal_decoder.py`. The `InstructionObject` is supposed to describe the instruction with the values for all the fields in the instruction. All decoding logic should be isolated to the plugin and not creep up in the `InstructionObject`.

Reply:
Done in 571c5525724b34f23a346e97628a1b1bc144be59